client/package.json

```diff
@@ -14,6 +14,7 @@
     "react": "^18.2.0",
     "react-bootstrap": "^2.10.0",
     "react-dom": "^18.2.0",
+    "react-drawio": "^0.1.2",
```

Comment:
Don't add react-drawio just yet, since nothing depends on it.

Reply:
Its okay. Something will depend on it in the nearest future.